base/gfsppieceop/pieceop.go

```diff
@@ -12,11 +12,11 @@ type GfSpPieceOp struct {
 }
 
 func (p *GfSpPieceOp) SegmentPieceKey(objectID uint64, segmentIdx uint32) string {
```

Comment:
@joeylichang  The objectID should be U256 and keep the same as the on-chain meta.

Reply:
BTW, It's better to use big-endian encoding for IDs here, as it ensures optimal performance during GetObject.